elasticsearch/install.sh

```diff
@@ -8,10 +8,20 @@ ln -s /usr/share/elasticsearch /usr/share/java/elasticsearch
 /usr/share/elasticsearch/bin/plugin install -b com.floragunn/search-guard-ssl/${SG_SSL_VER}
 /usr/share/elasticsearch/bin/plugin install -b com.floragunn/search-guard-2/${SG_VER}
 /usr/share/elasticsearch/bin/plugin install io.fabric8/elasticsearch-cloud-kubernetes/${ES_CLOUD_K8S_VER}
-/usr/share/elasticsearch/bin/plugin install io.fabric8.elasticsearch/openshift-elasticsearch-plugin/${OSE_ES_VER}
+#/usr/share/elasticsearch/bin/plugin install io.fabric8.elasticsearch/openshift-elasticsearch-plugin/${OSE_ES_VER}
+/usr/share/elasticsearch/bin/plugin install file:///openshift-elasticsearch-plugin-2.4.1.5-SNAPSHOT.zip
 
 mkdir /elasticsearch
 mkdir -p $ES_CONF
 chmod -R og+w $ES_CONF
 chmod -R og+w /usr/share/java/elasticsearch ${HOME} /elasticsearch
 chmod -R o+rx /etc/elasticsearch
+chmod +x /usr/share/elasticsearch/plugins/search-guard-2/tools/sgadmin.sh
+
+PASSWD=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 20 | head -n 1)
+cat > ${HOME}/sgconfig/sg_internal_users.yml << CONF
+---
+  $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 8 | head -n 1):
+    hash: $PASSWD
+CONF
+
```

Comment:
I think we will be good here.  Per the documentation, the hash is supposed to be 'bcrypt'.  I presume if someone ever tried to use it there would be some sort of error while unencrypting.  Regardless, I would prefer to see some changes to SG which ignored this file if it was not configured for it.